async_service/__init__.py

```diff
@@ -14,6 +14,10 @@
     SQSOutputConfig,
     WorkerConfig,
 )
+from async_service.function_service import (
+    create_async_server_for_functions,
+    create_worker_for_functions
```

Comment:
Do these need to be exposed to user?

Reply:
these were the two functions we thought we will expose to the user.